test/e2e/tests/smart-transactions/smart-transactions.spec.ts

```diff
@@ -157,7 +157,7 @@ describe('Smart Transactions', function () {
     );
   });
 
-  it('should execute a dApp Transaction', async function () {
+  it.skip('should execute a dApp Transaction', async function () {
```

Comment:
In the meantime this test is covered by this manual test scenario: https://github.com/MetaMask/metamask-user-journeys/blob/main/features/Trade/Swap/dapp-swap-STX-enable.feature